### PR TITLE
fsutil.get_path_usage

### PR DESCRIPTION
**Pull request by drmingdrmer**
```
fsutil.get_path_usage(path)

{
    'total':     total space in byte,
    'used':      used space in byte(includes space reserved for super user),
    'available': total - used,
    'percent':   float(used) / 'total',
}
```

### CODE REVIEW

File: fsutil/README.md [hunk 1]
```diff
@@ -205,6 +206,43 @@ Return the name of file system the `path` is on.
...
+{
+    'total':     total space in byte,
+    'used':      used space in byte(includes space reserved for super user),
+    'available': total - used,
+    'percent':   float(used) / 'total',
+}
+```
+
+There two concept for unused space: `free` and `available`
+because some file system has a reserved(maybe 5%) for super user like `root`:
```
Comment (wenbobuaa) [inline fsutil/README.md]: 没加复数形式的地方感觉怪怪的，some file systems have a reserved(maybe 5%) for super users

File: fsutil/README.md [hunk 2]
```diff
@@ -205,6 +206,43 @@ Return the name of file system the `path` is on.
...
+
+- free:      with    blocks reserved for super users.
+
+- available: without blocks reserved for super users.
+
+Since most time an application can not run as `root`
```
Comment (wenbobuaa) [inline fsutil/README.md]: most of the time

File: fsutil/README.md [hunk 3]
```diff
@@ -205,6 +206,43 @@ Return the name of file system the `path` is on.
 the file-system name, such as `ext4` or `hfs`.
 
 
+##  fsutil.get_path_usage
+
+**syntax**:
+`fsutil.get_path_usage(path)`
+
+Collect space usage information of the file system `path` is mounted on.
```
Comment (shuoqingding) [inline fsutil/README.md]: of the device where the `path` is mounted ?

File: fsutil/fsutil.py [hunk 4]
```diff
@@ -83,6 +83,25 @@ def get_path_fs(path):
     return prt_by_mp[mp]['fstype']
 
 
+def get_path_usage(path):
+
+    space_st = os.statvfs(path)
+
+    # f_bavail: without blocks reserved for super users
+    # f_bfree:  with    blocks reserved for super users
```
Comment (shuoqingding) [inline fsutil/fsutil.py]: `f_bfree` 这里没用到，是不是需要注释 `f_blocks`呢？

Comment (drmingdrmer) [reply fsutil/fsutil.py]: 是没用到, 啥叫 需要注释 f_blocks

**Pull request merged**